ext-apiserver/cloud/google/machineactuator.go

```diff
@@ -672,3 +676,13 @@ func getSubnet(netRange clusterv1.NetworkRanges) string {
 	}
 	return netRange.CIDRBlocks[0]
 }
+
+// TODO: We need to change this when we create dedicated service account for apiserver/controller
+// pod.
+//
+func CreateExtApiServerRoleBinding() error {
+	err := run("kubectl", "create", "rolebinding",
```

Comment:
(nit)

Is there any value to creating the err variable? I would imagine the following as cleaner.
return run(...)

Reply:
done